packages/kbn-ui-shared-deps/entry.js

```diff
@@ -36,6 +36,7 @@ export const React = require('react');
 export const ReactDom = require('react-dom');
 export const ReactDomServer = require('react-dom/server');
 export const ReactIntl = require('react-intl');
+export const ReactMarkdown = require('react-markdown');
```

Comment:
@spalger any issues with adding this to shared deps? It pulls in a decently heavy library (remark-parse) along with it in 5 different bundles. It's always going to be needed too since we use it in Core right now.

Reply:
Well, `ui-shared-deps` is really only for modules which must only have a single instance for state-management purposes. I'm worried that putting things in here as an optimization is a slippery slope, especially since putting things in the core bundle should be more than enough to reduce the module count (as long as people are importing from core and not importing `react-markdown` directly).

Ultimately though, I think this will work fine as long as we're okay with `react-markdown` being locked to a single version.

Reply:
Changed to use an async import instead for now.